src/migrations/migration.rs

```diff
@@ -126,13 +132,30 @@ async fn _read_names(dir: &Path) -> anyhow::Result<Vec<PathBuf>> {
         {
             continue;
         }
+
+        if !has_old_filename && old_name_re.is_match(item.file_name().to_str().unwrap()) {
```

Comment:
```suggestion
        if !has_old_filename && old_name_re.is_match(lossy_name) {
```

We just unpack the name a few lines up.